src/acquisition/flusurv/flusurv.py

```diff
@@ -49,7 +51,7 @@
 
 # all currently available FluSurv locations and their associated codes
 # the number pair represents NetworkID and CatchmentID
-location_codes = {
+location_to_code = {
```

Comment:
might we have to update this by hand in the future?  is there a programatic way to get this mapping?

Reply:
Yeah, this can be constructed from the metadata JSON, but would involve passing this obj around a bit awkwardly. Could be a point in favor of a metadata class.